pages/about.html

```diff
@@ -0,0 +1,170 @@
+          </div>
+
+          <div class="about__logo-image">
+            <img src="../images/logo.svg" alt="logo" />
+          </div>
+        </div>
+      </section>
+
+      <section id="news" class="about__past section">
+        <div class="container container-pall">
+          <h2 class="section-title">See the past Africa's Leading Tech Conferences</h2>
+
+          <p class="about__past-description">
+            Take a look at the last two Africa's Leading Tech Conferences which took place in Kigali, Rwanda and Cape
+            Town, South Africa.
+          </p>
+
+          <div class="about__past-conferences flex flex-collumn">
+            <a href="https://github.com/Timbar09" target="_blank" class="about__past-conference">
+              <img src="../images/2016-conference.jpg" alt="Photo from the 2016 conference" />
+              <div class="about__past-conference-overlay">
+                <div>
+                  <h3 class="about__past-sub-title sectio-sub-title">2016</h3>
+
+                  <p>AL Tech Conference 2016 in Kigali</p>
+                </div>
+              </div>
+            </a>
+
+            <a href="https://github.com/Timbar09" target="_blank" class="about__past-conference">
+              <img src="../images/2018-conference.jpg" alt="Photo from the 2018 conference" />
+              <div class="about__past-conference-overlay">
+                <div>
+                  <h3 class="about__past-sub-title sectio-sub-title">2018</h3>
+
+                  <p>AL Tech Conference 2018 in Cape Town</p>
+                </div>
+              </div>
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <section id="partners" class="partner section">
+        <div class="container container-pall">
+          <h2 class="partner__title section-title">Partners</h2>
+          <div class="partner__grid flex flex-jc-c flex-ai-c flex-wrap">
+            <img src="../images/mozilla-logo.svg" alt="Mozilla logo" class="partner__logo" />
+
+            <img src="../images/google-log.svg" alt="Google logo" class="partner__logo" />
+
+            <img src="../images/naver-logo.svg" alt="Naver logo" class="partner__logo" />
+
+            <img src="../images/daumkakao-logo.svg" alt="Daumkakao logo" class="partner__logo" />
+
+            <img src="../images/airbnb-logo.svg " alt="Airbnb logo" class="partner__logo" />
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer section">
+      <div class="container container-pall flex flex-collumn flec-jc-c flex-ai-c">
+        <img class="footer__logo" src="../images/logo-dark.svg" alt="logo" />
+
+        <div class="footer__info">
+          <p class="footer__copyright">2023 Africa's Leading Tech Conference. Some rights reserved</p>
+          <p class="footer__credits"><a class="about__link" href="https://github.com/Timbar09">Credits</a></p>
+        </div>
+      </div>
+    </footer>
+  </body>
+</html>
```

Comment:
![image](https://user-images.githubusercontent.com/60612329/210953816-9894339d-a072-479d-9d13-ce79ada29847.png)
your design ⬆️ 

![image](https://user-images.githubusercontent.com/60612329/210953993-6fd1542e-da66-444c-8d01-3723ac68032a.png)
required design ⬆️ 

are required your about section should not contain partners' information which is already present on the home page and your footer should have a background colour of black.

Reply:
Got it.

Reply:
Fixed.